### DISCUSSION TITLE
iFan03 Incorrect work

### DISCUSSION THREAD

**Discussion by fiska1** [RavenSystem/esp-homekit-devices#2007] [MEPLHAA Scripts]
**Detailed steps to reproduce it**

**MEPLHAA Script**

**Used device** 

**HAA version**

**iOS/iPadOS version**

**Logs**
A .txt file with logs. Read [how to capture logs from your device](https://github.com/RavenSystem/esp-homekit-devices/wiki/Build-instructions#capture-logs)

**Screen-shots or videos showing issue**

**Comment by fiska1** [comment]
Wrong job!!! Configured from the remote, the speeds work correctly. When using the remote control, the 1st speed is 3333%; the 2nd speed is 2000%; the 3rd speed is 1000%. When working in Home Kit 33% 67% and 100% !!! The hood immediately turns on to the maximum and is not regulated! Please fix the work in JSON

**Comment by RavenSystem** [comment]
I don't understand you, and more information is needed. No logs and no script provided.

**Comment by RavenSystem** [comment]
Moved to discussion because it is not a firmware bug.

**Comment by RavenSystem** [comment]
Duplicated is issues, but it is not an issue. Please, use issues only for issues.
